src/main/java/zd/zero/waifu/motivator/plugin/assets/AssetModels.kt

```diff
@@ -10,18 +15,23 @@ data class ImageDimension(
 )
 
 data class VisualMotivationAssetDefinition(
-    val imagePath: String,
+    val imagePath: String, // todo: remove this once migrated
```

Comment:
I'll eventually create an issue to address this when this can be worked. I have a couple reminders.